src/applications/vaos/components/VAFacilityInfoMessage.jsx

```diff
@@ -3,20 +3,23 @@ import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
 export default function VAFacilityInfoMessage({ facility }) {
   return (
-    <AlertBox status="info" headline="We found one VA location for you">
-      <p>
-        <strong>{facility.institution.authoritativeName}</strong>
-        <br />
-        {facility.institution.city}, {facility.institution.stateAbbrev}
-      </p>
-      Not all VA locations offer all types of care or support online scheduling.
-      <p>
-        If this location wasn't what you were looking for, you can{' '}
-        <a href="/find-locations" target="_blank" rel="noopener noreferrer">
-          search for a nearby location
-        </a>{' '}
-        and call to schedule an appointment.
-      </p>
-    </AlertBox>
+    <div aria-atomic="true" aria-live="assertive">
```

Comment:
I don't think we want to add these to this message. It's informational and is the primary page content.

Reply:
@jbalboni, i'll revert VAFacilityInfoMessage.jsx back to previous version and remove VAFacilityInfoMessage.unit.spec.jsx from the commit.  Is there any others that need to revert back?

Reply:
@jenniemc Nope, just this one.

Reply:
@jbalboni, this commit failed the check in the Jenkin pipeline at Integration. Can you take a look why it failed at Accessibility?